shiny/global-sensitivity/server.R

```diff
@@ -0,0 +1,72 @@
+library(shiny)
+library(ncdf4)
+library(ggplot2)
+
+source("helper.R")
+source("plotEnsemble.R")
+source("load_ensemble.R")
+
+# Define server logic
+server <- shinyServer(function(input, output, session) {
+    bety <- betyConnect()
+    logger.setLevel("OFF")
+    ranges <- reactiveValues(x = NULL, y = NULL)
+    
+    print("RESTART")
+    
+    # set the workflow id(s)
+    ids <- get_workflow_ids(bety, session)
+    default_workflow_id <- 1000001488    # Working default
```

Comment:
Filter by runtype is implemented. Number of ensemble members, I still need to make.